src/_shared/js/ads.js

```diff
@@ -11,17 +11,25 @@ const GLABS_EDITION = {
     US: 'guardian-labs-us'
 };
 
-function addTrackingPixel(rootNode) {
+function addTrackingPixel() {
     if( !trackingPixel ) return;
+    loadImage(trackingPixel);
+}
+
+function addPixel(url) {
+    loadImage(url);
+}
 
-    if( !pixelTemplate ) {
-        pixelTemplate = document.createElement('img');
-        pixelTemplate.className = 'creative__pixel';
-    }
+function addViewabilityTracker(creativeId) {
+    var tracker = document.getElementById('viewabilityTracker');
+    if( !tracker ) return;
+    tracker = tracker.firstChild.nodeValue.trim();
```

Comment:
Very good question. The only idea I had to get this working was to use a construct such as this: `<div><!-- [%Viewabilitytracker%] --></div>`, i.e. a mini DOM tree with only one comment node containing the content of the `Viewability tracker` DFP variable. Then I can reify the content of the comment, using the method above (yes, `firstChild` will always exist, but it may be empty).

Come to think of it, is this extra step really necessary? We could instead do this in the DFP tempate:

`[%Viewabilitytracker%]`

... the only problem is the `UNIQUE_ID`. I'm not sure it is even useful. What do you think?